hera_mc/weather.py

```diff
@@ -0,0 +1,226 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Handling quality metrics of HERA data.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.time import Time
+from math import floor
+from sqlalchemy import Column, BigInteger, Float, String
+
+import tornado.gen
+
+from . import MCDeclarativeBase
+
+weather_sensor_dict = {'wind_speed': {'sensor_name': 'anc_mean_wind_speed',
```

Comment:
Is this where we add if we want to get more sensors?

Reply:
Yes. You should also add their details to docs/weather_sensor_details.txt (you can get that using their example scripts).